charts/prometheus-blackbox-exporter/templates/servicemonitor.yaml

```diff
@@ -30,6 +30,10 @@ spec:
         replacement: {{ .url }}
       - targetLabel: target
         replacement: {{ .name }}
+        {{- range $targetLabel, $replacement := .additionalMetricsRelabels }}
```

Comment:
Should we have defaults as well?

Reply:
That should be set with https://github.com/prometheus-community/helm-charts/pull/69/files#diff-711ccd42b755d858f62f44246a271e46R132 ?

Reply:
Oh, wait, I see what you mean.